testing_exercise/testing.js

```diff
@@ -0,0 +1,41 @@
+function replaceWith(str, target, replacement){
+	return str
+		.split('')
+		.map(function(v,i,a){
+			if(v === target) return replacement;
+			else return v;
+		}).join('');
+}
+
+function expand(arr, num) {
+	var a = [];
+
+	for(let i = 0; i < num; i++){
+		a = a.concat(arr);
+	}
+
+	return a;
+}
+
+function acceptNumbersOnly(...args) {
+	for(let a of args){
+		if(typeof a !== 'number' || isNaN(a)){
+			return false;
+		}
+	}
+
+	return true;
+}
+
+function mergeArrays(arr1, arr2) {
+	return arr1.concat(arr2).sort();
```

Comment:
one gotcha to be aware of with `sort` - it won't sort numbers correctly! The default sort in JavaScript is alphabetical, so for example:

`[23,1,4].sort() // [1, 23, 4]`

The example doesn't catch this. 

To fix this, you can pass in a comparator to the sort function telling it how it should sort. [reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort)